_pages/documentation/index.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: documentation
+title: gem5 documentation
+permalink: /documentation/
+author: Jason Lowe-Power
+---
+
+# gem5 Documentation
+
+## Learning gem5
+
+[Learning gem5](learning_gem5/introduction/) gives a prose-heavy introduction to using gem5 for computer architecture research written by Jason Lowe-Power.
+This is a great resource for junior researchers who plan on using gem5 heavily for a research project.
+
+It covers details of how gem5 works starting with [how to create configuration scripts](learning_gem5/simple_config).
+It then goes on to describe [how to modify and extend](learning_gem5/part2) gem5 for your research including [creating `SimObjects`](learning_gem5/part2/helloobject), [using gem5's event-driven simulation infrastructure](learning_gem5/part2/events), and [adding memory system objects](learning_gem5/part2/memoryobject).
+In [Learning gem5 Part 3](learning_gem5/part3) the [Ruby cache coherence model](/documentation/memory_systems/ruby) is discussed in detail including a full implementation of an MSI cache coherence protocol.
```

Comment:
"Learning gem5 Part 3" leads to a very basic index page, which I'm not sure was the original intention: 
![Screenshot from 2019-11-21 10-00-53](https://user-images.githubusercontent.com/43390222/69363733-dc81eb00-0c45-11ea-9405-8bb5dd71c792.png)

Ditto the "How to modify and extend" above.

Reply:
Also the "Ruby cache coherence model" link is also broken

Reply:
That link will be fixed in a follow on patch

Reply:
Also, I think I'll keep this basic index page for now. I'll come back (someday) and fill in intros to each section.

Reply:
The "Ruby cache coherence model" link is still broken. Outside of this I'm happy